src/chapters/15-part-2-recap.md

```diff
@@ -0,0 +1,35 @@
+Congratulations, you finished the second part of the tutorial!
+
+There was a lot of concent to cover in part two. To recap, here is what you learned:
+
+<h3><a href="../11-route-params/">Chapter 11</a></h3>
+
+* Routes with dynamic segments
+* Links with dynamic segments
+* Component tests with access to the router
+* Accessing parameters from dynamic segments
+* Sharing common setup code between tests
+
+<h3><a href="../12-service-injection/">Chapter 12</a></h3>
+
+* Splattributes and the `class` attribute
+* The router service
+* Ember services vs. global variables
+* Mocking services in tests
+
+<h3><a href="../13-ember-data/">Chapter 13</a></h3>
+
+* Ember Data models
+* Testing models
+* Loading models in routes
+* The Ember Data store
+* Working with adapters and serializers
+
+<h3><a href="../14-provider-components/">Chapter 14</a></h3>
+
+* Using Ember's built-in `<Input>` helper
+* The provider component pattern
+* Using block parameters when invoking components
+* Yielding data to caller components
+
+Awesome! The concepts you learned about in part 2 of the tutorial are ones that you'll find in many production-level Ember apps. You've now taken your knowledge to the next level&mdash;and you've also finished the entire tutorial, hooray!
```

Comment:
@chancancode I feel like there's another sentence or two that belongs here...but I'm not sure what to say?